website/src/pages/HomePage/components/HomePageHero.tsx

```diff
@@ -0,0 +1,118 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Link, To } from "react-router-dom";
+import Icon from "@/common/Icon";
+import { Button } from "@/components/ui/button";
+import DailySummary from "@/common/DailySummary";
+import useDailySummaryData from "@/hooks/useDailySummaryData";
+import { useState } from "react";
+
+export default function HomePageHero() {
+  const [benchmarkType, setBenchmarktype] = useState<string>("");
+
+  function getBenchmarkType(type: string) {
+    setBenchmarktype(type);
+  }
+
+  const {
+    dataDailySummary,
+  } = useDailySummaryData();
+
+  return (
+    <section className="flex flex-col items-center h-screen p-page">
+      <h1 className="text-6xl font-semibold text-center mt-10 leading-normal">
+        Benchmarking <br />
+        System for <br />
+        <span className="text-orange-500"> Vitess</span>
+      </h1>
+      <div className="flex gap-x-4 mt-10">
+        <Button
+          asChild
+          size={"lg"}
+          variant={"default"}
+          className="bg-background hover:bg-muted/90 dark:bg-front dark:hover:bg-front/90 dark:text-background text-foreground rounded-lg border"
+        >
+          <Link
+            className="rounded-2xl p-5 flex items-center gap-x-2"
+            to="https://vitess.io/blog/2021-07-08-announcing-vitess-arewefastyet"
+            target="__blank"
+          >
+            Blog Post
+            <Icon className="text-2xl" icon="bookmark" />
+          </Link>
+        </Button>
+        <Button
+          asChild
+          size={"lg"}
+          variant={"default"}
+          className="bg-front text-background hover:bg-front/90 rounded-lg"
+        >
+          <Link
+            className="rounded-2xl p-5 flex items-center gap-x-2"
+            to="https://github.com/vitessio/arewefastyet"
+            target="__blank"
+          >
+            GitHub
+            <Icon className="text-2xl" icon="github" />
+          </Link>
+        </Button>
+        <Button
+          asChild
+          size={"lg"}
+          variant={"default"}
+          className="bg-background hover:bg-muted/90 dark:bg-front dark:hover:bg-front/90 dark:text-background text-foreground rounded-lg border"
+        >
+          <Link
+            className="rounded-2xl p-5 flex items-center gap-x-2"
+            to="https://www.vitess.io"
+            target="__blank"
+          >
+            Vitess
+            <Icon className="text-2xl" icon="vitess" />
+          </Link>
+        </Button>
+      </div>
+      <h2 className="text-2xl font-medium mt-20">
+        Historical results on the <Link className="text-primary" to="https://github.com/vitessio/arewefastyet/tree/main" target="_blank">main</Link>{" "}
```

Comment:
This should point to vitessio/vitess's main branch.

Reply:
I think this is already pointing to vitessio/vitess's main with the following url `https://github.com/vitessio/arewefastyet/tree/main`

Reply:
I meant to point to the `main` branch of the vitess repository not the arewefastyet repository, the link should be: https://github.com/vitessio/vitess/tree/main/